addons/mail/models/mail_alias.py

```diff
@@ -171,6 +174,38 @@ def open_parent_document(self):
             'type': 'ir.actions.act_window',
         }
 
+    def _get_alias_bounced_body_fallback(self, message_dict):
+        return _("""Hi,<br/>
+The following email sent to %s cannot be accepted because this is a private email address.
+Only allowed people can contact us at this address.""" % message_dict.get('to'))
+
+    def _get_alias_bounced_body(self, message_dict):
+        """Get the body of the email return in case of bounced email.
+
+        :param message_dict: dictionary of mail values
+        """
+        lang_author = False
+        if message_dict.get('author_id'):
+            try:
+                author = self.env['res.partner'].browse(message_dict['author_id'])
+            except:
+                pass
```

Comment:
Please no.
What can fail in the browse call?

Reply:
Indeed, check of lang has been added.

Reply:
Even. bare excepts are a bad pattern. and even more with a `pass` inside.
Why can't you use `sudo()` here?